src/lib.rs

```diff
@@ -204,6 +206,30 @@ pub fn create_ecx<'a, 'mir: 'a, 'tcx: 'mir>(
         }
     }
 
+    // Cache some data used by unwinding
+    if ecx.tcx.tcx.sess.panic_strategy() == PanicStrategy::Unwind {
+
+        // We build up the layout of '*mut &mut dyn core::panic::BoxMeUp'
+        let box_me_up_did = ecx.resolve_did(&["core", "panic", "BoxMeUp"])?;
```

Comment:
Seems a bit strange to hard-code a path here... does the LLVM codegen backend do the same?

Reply:
[libpanic_unwind](https://github.com/rust-lang/rust/blob/00856722bad5e9d96048319fb41f4b7e249820cd/src/libpanic_unwind/lib.rs#L100) does

Reply:
Oh wow, that's some crazy stuff. Please add a reference to that in the code.